README.md

```diff
@@ -153,93 +186,92 @@ This can be used to represent all kinds of errors. It can be for errors that hap
+| <a name="event.severity"></a>`event.severity`  | Severity of the event. Severity values can very different across use cases. The implementer should make sure that severities are consistent across events.  | long  |   | `7`  |
+| <a name="event.raw"></a>`event.raw`  | Raw text message of entire event. Used to demonstrate log integrity.  | keyword  |   | `Sep 19 08:26:10 host CEF:0&#124;Security&#124; threatmanager&#124;1.0&#124;100&#124; worm successfully stopped&#124;10&#124;src=10.0.0.1 dst=2.1.2.2spt=1232`  |
 | <a name="event.hash"></a>`event.hash`  | Hash (perhaps logstash fingerprint) of raw field to be able to demonstrate log integrity.  | keyword  |   | `123456789012345678901234567890ABCD`  |
-| <a name="event.version"></a>`event.version`  | The version field contains the version an event for ECS adheres to.<br/>This field should be provided as part of each event to make it possible to detect to which ECS version an event belongs.<br/>event.version is a required field and must exist in all events. It describes which ECS version the event adheres to.<br/>The current version is 0.1.0.  | keyword  |   | `0.1.0`  |
+| <a name="event.version"></a>`event.version`  | The ECS version that an event adheres to.<br/>Required field for all events. <br/> [Current version](#ecs-version) | keyword  |   | `0.1.0`  |
 | <a name="event.duration"></a>`event.duration`  | Duration of the event in nanoseconds.  | long  |   |   |
-| <a name="event.created"></a>`event.created`  | event.created contains the date when the event was created.<br/>This timestamp is distinct from @timestamp in that @timestamp contains the processed timestamp. For logs these two timestamps can be different as the timestamp in the log line and when the event is read for example by Filebeat are not identical. `@timestamp` must contain the timestamp extracted from the log line, event.created when the log line is read. The same could apply to package capturing where @timestamp contains the timestamp extracted from the network package and event.created when the event was created.<br/>In case the two timestamps are identical, @timestamp should be used.  | date  |   |   |
+| <a name="event.created"></a>`event.created`  | Date the event was created.<br/>**TBD.** @timestamp and event.created both use "Time the event was created" as the description. Can we further clarify by saying WHO is doing the creating of each?<br/><br/>This field is distinct from @timestamp, even though the values may be the same in some cases.<br/>Examples: For logs, `@timestamp` contains the timestamp carried in the log line, and `event.created` contains the time when the log line is read. For package captures, `@timestamp` contains the timestamp extracted from the network package, and `event.created` contains the time the event was created. </br>If the two timestamps are identical, use [@timestamp](#timestamp). | date  |   |   |
```

Comment:
+1 on making the destinction very clear. I hoped the examples help here.

Reply:
Examples helped for sure!  Adding a few more (or slightly different) words describing "who" or "when" would help even more to differentiate. I tried "Data/time the event originated" for @timestamp. Is that accurate?